Azure.ResourceManager.Core/AzureResourceManagerClient.cs

```diff
@@ -71,17 +71,17 @@ public AzureResourceManagerClient(Uri baseUri, TokenCredential credential, strin
 
         internal virtual AzureResourceManagerClientOptions ClientOptions { get; }
 
-        public SubscriptionOperations Subscription(SubscriptionData subscription) => new SubscriptionOperations(ClientOptions, subscription);
+        public SubscriptionOperations GetSubscriptionOperations(SubscriptionData subscription) => new SubscriptionOperations(ClientOptions, subscription);
 
         /// <summary>
         /// </summary>
         /// <param name="subscription"></param>
         /// <returns></returns>
-        public SubscriptionOperations Subscription(ResourceIdentifier subscription) => new SubscriptionOperations(ClientOptions, subscription);
+        public SubscriptionOperations GetSubscriptionOperations(ResourceIdentifier subscription) => new SubscriptionOperations(ClientOptions, subscription);
 
-        public SubscriptionOperations Subscription(string subscription) => new SubscriptionOperations(ClientOptions, subscription);
+        public SubscriptionOperations GetSubscriptionOperations(string subscription) => new SubscriptionOperations(ClientOptions, subscription);
```

Comment:
We should remove this overload since there is an implicit conversion from string -> ResourceIdentifier